app/javascript/components/UI/rich-text/rich-text.js

```diff
@@ -0,0 +1,165 @@
+
+export default class RichText {
+  
+  static filtrationById = ( separatorArr, mainArr ) => {
+    if ( !separatorArr.length ) {return mainArr}
```

Comment:
are you sure !separatorArr.length works out for you here for all cases?

Reply:
Thanks. Yes of course I've been checking it and thanks to the operator '!' in JS is working so like this  !null = true, !0 = true, !undefined = true

Reply:
but you are not checking only null here. Can you get length prop of **!null** ??